### PR TITLE
Add selected for plotting indicator to experiments webview

### PR DESCRIPTION
**Pull request by mattseddon**
Based on the conversation we had in the planning meeting regarding how we link filled circles with the fact that those experiments have been selected for plotting. 

From my understanding, this was one of the @sroy3 ideas.

We can display an indicator with the graph icon that shows how many experiments are selected for plotting. 

We can also provide an action to open the plots webview:

https://user-images.githubusercontent.com/37993418/179870922-1750d7a0-a19f-4f6f-b356-8776afc29d75.mov

~Code is only a prototype and needs to be restructured.~

What does everyone think?

@sroy3 @wolmir @julieg18 @shcheklein @maxagin

### CODE REVIEW

**Comment by maxagin** [comment]
Very good idea, I think. Can we split the screen when the icon clicks, so the user will have the context?

**Review by julieg18 — APPROVED** [review]
Looks great!

File: webview/src/experiments/components/table/Indicators.tsx [hunk 1]
```diff
@@ -115,19 +123,54 @@ const formatFilteredCountMessage = (filteredCounts: FilteredCounts): string =>
...
 export const Indicators = ({
+  rows,
   sorts,
   filters,
   filteredCounts
 }: {
   sorts?: SortDefinition[]
   filters?: string[]
   filteredCounts: FilteredCounts
+  rows: Row<Experiment>[]
```
Comment (mattseddon) [inline webview/src/experiments/components/table/Indicators.tsx:153]: [C] I don't think I should be giving this access to every single row. I will update to calculate the `selectedForPlotsCount` higher up.

**Comment by mattseddon** [comment]
> Very good idea, I think. Can we split the screen when the icon clicks, so the user will have the context?

IMO it's less opinionated to just open the view. The user can make the decision on how to position everything inside of their IDE for themselves. Let's see if users want this/ask them before implementing that change.

**Pull request merged**